lib/command.c

```diff
@@ -42,11 +42,80 @@
 #include "command_graph.h"
 #include "qobj.h"
 #include "defaults.h"
+#include "termtable.h"
 
 DEFINE_MTYPE(LIB, HOST, "Host config")
 DEFINE_MTYPE(LIB, STRVEC, "String vector")
 DEFINE_MTYPE(LIB, COMPLETION, "Completion item")
 
+const char *node_names[] = {
```

Comment:
Regarding adding documentation, I already did that.

For the second part I already did that too. I dropped it for a couple reasons. One is that the node names will be needed in other places. By making the names a parallel array you can get the name of a node just by using the `node_type` as an array index. Putting the names into the struct definition itself means you can't get the names unless you access to the struct, and almost all of them are static. It's cleaner and easier to do it this way.